Scroller/EditableWrapper.js

```diff
@@ -364,14 +371,16 @@ const EditableWrapper = (props) => {
 
 	useEffect(() => {
 		// Calculate the item width once
-		if (!mutableRef.current.itemWidth) {
-			const item = wrapperRef.current?.children[0];
+		const scrollContentNode = scrollContentRef.current;
+		const item = scrollContainerHandle.current.rtl ? wrapperRef.current?.children[dataSize - 1] : wrapperRef.current?.children[0];
+		if (item) {
 			const neighbor = item.nextElementSibling || item.previousElementSibling;
+
 			mutableRef.current.itemWidth = Math.abs(item.offsetLeft - neighbor?.offsetLeft);
-			mutableRef.current.centeredOffset = centered ? item.getBoundingClientRect().x : 0;
+			mutableRef.current.centeredOffset = (centered && (scrollContentNode.getBoundingClientRect().width > mutableRef.current.itemWidth * dataSize)) ? item.getBoundingClientRect().x : 0;
```

Comment:
Why we need this new condition? please leave the reason.

Reply:
In LTR, centeredOffset set with the value of mutableRef.current.child[0].getBoudningClientRect().x 
But in RTL, centeredOffset cannot be figured out with the value of getBoundingClientRect().x as shown in the picture below.
![20220518_152733_centered_offset](https://user-images.githubusercontent.com/77044692/168973188-1fa4c34e-e2a6-4f40-8236-b2c90c80175a.png)
We need the value of green line to set centeredOffset instead of red line(getBoundingClientRect().x).